geoutils/georaster.py

```diff
@@ -656,21 +657,21 @@ def crop(self, cropGeom, mode='match_pixel'):
             resolution, cropping to the extent that most closely aligns with the current coordinates. 'match_extent'
             will match the extent exactly, adjusting the pixel resolution to fit the extent.
         :type mode: str
-
+        :param inplace: Update the raster inplace or return copy.
+        :type inplace: bool
+        :returns: None if inplace=True and a new Raster if inplace=False
         """
         import geoutils.geovector as vt
 
         assert mode in ['match_extent', 'match_pixel'], "mode must be one of 'match_pixel', 'match_extent'"
-        if isinstance(cropGeom, Raster):
+        if isinstance(cropGeom, (Raster, vt.Vector)):
```

Comment:
Have you tested this? The vector objects currently don't have an attribute bounds I believe. I always call the underlying geopandas attribute `self.ds.total_bounds`. (I know, we should make Vector more consistent with Raster...)

Reply:
Haaa, I see you've added it below! ;-)